packages/yandex-metrika/plugin.js

```diff
@@ -7,11 +7,15 @@ export default ({ app: { router } }) => {
   })
 
   function create() {
-    ym(<%= options.id %>, "init", <%= JSON.stringify(options) %>);
+
     router.afterEach((to, from) => {
       if (!ready) {
         // Don't record a duplicate hit for the initial navigation.
-        return
+        (function(m,e,t,r,i,k,a){m[i]=m[i]||function(){(m[i].a=m[i].a||[]).push(arguments)};
```

Comment:
I think this is a wrong place to init script (notice `router.afterEach`)

Reply:
You're right. I'm fixed it.